### ISSUE TITLE
Displaying onboarding status on ChA participant admin page

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4744]
Connected to #4743 

Add a section to the ChA participant admin page with the four onboarding steps that will track which steps that ChA has completed.

**Comment by shaun-technovation** [comment]
This is on Preview now.

**Comment by dboyer** [comment]
Confirming on Preview that I see the onboarding section with checkmarks for the four required tasks. I confirmed that the steps do update when each is completed.

I saw the background check status notes appear differently for different ChAs depending on their status (complete, expire,d invite not sent, etc). I think we can keep that section for previous ChAs for now and remove it later if it becomes confusing.

<img width="695" alt="Screen Shot 2024-06-02 at 11 26 24 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/42997ab7-21a4-4649-9ba5-d6a7e9d4369a">

<img width="590" alt="Screen Shot 2024-06-02 at 11 29 00 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/757cfbe8-e738-49d3-ae01-ce8eef29877a">
